core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -4955,7 +4955,7 @@ LogicalProject(EXPR$0=[1])
       LogicalProject(DEPTNO=[$7])
         LogicalFilter(condition=[>($7, 10)])
           LogicalTableScan(table=[[CATALOG, SALES, EMP]])
-    LogicalFilter(condition=[>($7, 7)])
+    LogicalFilter(condition=[OR(>($7, 7), >($7, 10))])
       LogicalTableScan(table=[[CATALOG, SALES, EMP]])
 ]]>
```

Comment:
`OR(>($7, 7), >($7, 10))` cannot be simplified to `>($7, 7)`?

Reply:
it could be; but this logic may not declare an inequality as a predicate - as that could introduce oversimplifications